lib/net/ssh/multi/session.rb

```diff
@@ -447,7 +450,7 @@ def io_select_wait(wait)
 
     def keepalive_interval
       servers = server_list.select { |s| s.options[:keepalive] }
-      servers.map { |s| s.options[:keepalive_interval] }.compact.min
+      servers.map { |s| s.options[:keepalive_interval] || DEFAULT_IO_SELECT_TIMEOUT }.compact.min
```

Comment:
If we make [the `keepalive` attribute on Net::SSH::Connection::Session](https://github.com/net-ssh/net-ssh/blob/v3.1.1/lib/net/ssh/connection/session.rb#L83) public, we could do this:

```ruby
class Net::SSH::Multi::Session
  def keepalive_interval
    server_list.map { |s| s.keepalive_interval }.compact.min
  end
end

class Net::SSH::Multi::Server
  def keepalive_interval
    session.keepalive.interval if session && session.keepalive.enabled?
  end
end
```

Alternatively we could check `enabled?` in Net::SSH, and expose a `keepalive_interval` method:

```ruby
class Net::SSH::Connection::Session
  def keepalive_interval
    @keepalive.interval if @keepalive.enabled?
  end
end
```

The problem is writing tests; right now the Net::SSH::Multi test suite stubs out all interaction with Net::SSH, but to test the implementation described above the [Server](https://github.com/net-ssh/net-ssh-multi/blob/v1.2.1/lib/net/ssh/multi/server.rb) object has to have an active [Session](https://github.com/net-ssh/net-ssh/blob/v3.1.1/lib/net/ssh/connection/session.rb), which means [calling Net::SSH.start](https://github.com/net-ssh/net-ssh-multi/blob/v1.2.1/lib/net/ssh/multi/server.rb#L186), which will [try to open a socket](https://github.com/net-ssh/net-ssh/blob/v3.1.1/lib/net/ssh/transport/session.rb#L70-L71)...